sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/samples/SamplesDbConnector.groovy

```diff
@@ -35,6 +35,31 @@ class SamplesDbConnector implements CountSamplesDataSource {
         this.connectionProvider = connectionProvider
     }
 
+    /**
+     * {@inheritDoc}
+     * @since 1.0.0
+     * We need to fetch all samples initially, since only the latest status counts. This only remains true, while
+     * Status.DATA_AVAILABLE is the last status a sample can reach.
+     */
+    @Override
+    List<String> fetchSampleCodesWithData(String projectCode) {
+        String queryTemplate = Query.fetchLatestSampleEntries()
+        queryTemplate += "WHERE sample_status = ?"
```

Comment:
```suggestion
        queryTemplate += " WHERE sample_status = ?"
```
You need to add leading whitespace or join it with whitespace as a delimiter. Otherwise, the query is incorrect.